src/implementations/twig/components/search-form/search-form.story.js

```diff
@@ -0,0 +1,39 @@
+import { withKnobs, text, optionsKnob } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl/storybook-addon-notes';
+import { getExtraKnobs, tabLabels, getComplianceKnob } from '@ecl/story-utils';
+import withCode from '@ecl/storybook-addon-code';
+
+import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
+import dataDefault from '@ecl/specs-component-search-form/demo/data';
+import searchForm from './search-form.html.twig';
+import notes from './README.md';
+
+const prepareSearchForm = (data) => {
+  data.button.label = text(
+    'button.label',
+    data.button.label,
+    tabLabels.required
+  );
+  data.button.icon.path = optionsKnob(
+    'button.icon.path',
+    { current: defaultSprite, 'no path': '' },
+    defaultSprite,
+    { display: 'inline-radio' },
+    tabLabels.required
+  );
+
+  getExtraKnobs(data);
+  getComplianceKnob(data);
+
+  return data;
+};
+
+export default {
+  title: 'Components/Forms/Search Form',
+};
+
+export const Default = () => searchForm(prepareSearchForm(dataDefault));
```

Comment:
Strangely, https://ecl-preview-1760--europa-component-library.netlify.app/playground/ec/?path=/story/components-forms-search-form--default does not seem to look like https://ecl-twig-js.netlify.app/ec/?path=/story/components-forms-search-form--default 

The search label doesn't seem to show up

Reply:
It was the difference in addons panel. So the panel of responsive was below the breakpoint to get the same style.
Bref, no issues.